libs/ngx-charts-on-fhir/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@elimuinformatics/ngx-charts-on-fhir",
-  "version": "6.1.3",
+  "version": "6.1.4",
```

Comment:
need to use version 6.1.5 because the version bumped in my PR is 6.1.4.

Reply:
updated